dashboard/src/stores/__tests__/package.test.ts

```diff
@@ -2,144 +2,144 @@ import { createPinia, setActivePinia } from "pinia";
 import { beforeEach, describe, expect, it } from "vitest";
```

Comment:
I think this file should be renamed "ingest.test.ts"?  Maybe you do that in a subsequent commit?

Reply:
Ups, no, I just missed this one, thanks!